proto/typedb_clustering_service/typedb_clustering.proto

```diff
@@ -0,0 +1,25 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+syntax = "proto3";
+
+package typedb.protocol;
+
+import "proto/typedb_service/server.proto";
+
+message Registration {
```

Comment:
When does this message gets sent again, and by whom to whom? The reason I'm asking is because now I feel the name `Registration` might not be entirely describe what the message does.

Reply:
How does the "registration" relate to the "add node conf change" raft message? In terms of terminology, they're quite synonymous even though the latter is where the bulk of the logic for adding a server takes place.